spring-cloud-starter-common-security-config-web/pom.xml

```diff
@@ -7,21 +7,12 @@
 		<version>1.8.0-SNAPSHOT</version>
 	</parent>
 	<artifactId>spring-cloud-starter-common-security-config-web</artifactId>
-	<properties>
-		<spring-security-oauth2.version>2.5.1.RELEASE</spring-security-oauth2.version>
-	</properties>
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-common-security-config-web</artifactId>
 			<version>1.8.0-SNAPSHOT</version>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.security.oauth</groupId>
```

Comment:
Any impacts to removing this or was it just not needed [here](https://github.com/spring-cloud/spring-cloud-common-security-config/commit/9bcf040ea02a33016594b5c7092621946967a759#diff-ce1d7fb9235bc275e43356e6fb66ab3cc755175081c4912474ba22648b42c9daR20)?